document/static/js/es6_modules/editor/menus/content-dialogs/table.js

```diff
@@ -0,0 +1,83 @@
+import {tableSchema} from "prosemirror/dist/schema-table"
+const {Table, TableRow, createTable, addTableNodes} = require("prosemirror/dist/schema-table")
```

Comment:
I think this need to be rewritten as

import {Table, TableRow, createTable, addTableNodes} from "prosemirror/dist/schema-table"